docs/tutorials/using-data-sources.md

```diff
@@ -361,7 +356,7 @@ Vega provides a timestamp source, which is useful for terminating a market at a
 
 As the name implies, a built in data source is generated inside Vega, and cannot be submitted by other keys.
 
-### Using built-in oracle data for trading termination
+### Using built-in data for trading termination
```

Comment:
We should not that if using time we should always use greater than or equals rather than a straight equals, to avoid missing the time because no event is emitted with the precise required trimestamp.